examples/shared_samples/logical_interconnect.rb

```diff
@@ -266,7 +270,7 @@
 puts "\nOperations with scopes"
 begin
   # Scope class used in this sample
-  scope_class = OneviewSDK.resource_named('Scope', @client.api_version) unless @client.api_version.to_i <= 200
+  scope_class = OneviewSDK.resource_named('Scope', @client.api_version) unless @client.api_version.to_i <= 200 || client.api_version.to_i >= 600
```

Comment:
we have to modify this condition once we support 800 API. Its better to give if condition here

Reply:
As Discussed, Keeping as is.